CMakeLists.txt

```diff
@@ -90,6 +97,11 @@ know it to be required.]=] OFF)
 cmake_dependent_option(OPENVDB_DISABLE_BOOST_IMPLICIT_LINKING
   "Disable the implicit linking of Boost libraries on Windows" ON "WIN32" OFF)
 option(USE_CCACHE "Build using Ccache if found on the path" ON)
+option(USE_STATIC_DEPENDENCIES [=[
+Only search for and use static libraries. If OFF the shared versions of requried libraries are prioritised, falling
+back to static libraries. Forcing individual static dependencies can be enabled by setting XXX_USE_STATIC_LIBS
+to ON, where XXX is the package name. On Windows this behaviour is less strict, with any located libraries assumed
+to be static. ]=] OFF)
```

Comment:
It's important to define `USE_STATIC_DEPENDENCIES` on windows even if you only have static libs installed as it's not possible to infer library types from the .libs alone. This will get better as we start to deprecate our Find modules in favour of 3rd party configuration files - but as we still need to support older versions and different pack-manager solutions, there's no single available fix.

Note that this does not impact C Runtime selection but is necessary for OpenEXR (where `OpenEXR_DLL` must *not* be defined) and Boost (where both shared and static libs are installed on the CI server).